litgpt/pretrain.py

```diff
@@ -75,6 +77,7 @@ def setup(
             ``model_config``.
         out_dir: Directory in which to save checkpoints and logs. If running in a Lightning Studio Job, look for it in
             /teamspace/jobs/<job-name>/share.
+        precision: The precision to use for finetuning. Possible choices: "bf16-true", "bf16-mixed", "32-true".
```

Comment:
If you change the type annotation to use Literal[...], jsonargparse will show the choices in the docstring automatically and do input validation

Reply:
Sounds good!